new-pipeline/style/dashboards.css

```diff
@@ -43,6 +43,13 @@
     z-index: 1;
     top: 5px;
 }
+
+/** Max Width Setting For e10s **/
+#filter-e10s + .btn-group ul
+{
+    max-width: 10vw !important;
+}
+
```

Comment:
This code has already been reviewed and merged. Maybe rebase your working copy atop the latest from mozilla/telemetry-dashboard?